presidio-image-anonymizer/app.py

```diff
@@ -20,27 +23,38 @@ class Server:
     def __init__(self):
         self.logger = logging.getLogger("presidio-image-anonymizer")
         self.app = Flask(__name__)
+        self.swagger = Swagger(
+            self.app, template={"info": {"title": "Presidio Image Anonymizer API"}}
+        )
         self.engine = ImageAnonymizerEngine()
 
         @self.app.route("/health")
         def health() -> str:
-            """Return basic health probe result.  get ok + 200."""
+            """Return basic health probe result.
+
+            ---
+            responses:
+              200:
+                description: OK
+                content:
+                  text/plain:
+                    schema:
+                      type: string
+                      example: ok
+            """
             return "ok"
```

Comment:
if we're changing the healthcheck, lets return a nicer string. "Presidio Analyzer/Anonymizer/Image-Reducter Service is Up"

Reply:
Done, currently leaving 'image anonymizer', we'll do the change in batch